README.md

```diff
@@ -60,19 +67,19 @@ The specification below describes the `trigger` metadata in `ScaledObject` resou
   triggers:
     - type: external
       metadata:
-        scalerAddress: keda-cosmosdb-scaler.keda:4050 # Mandatory. Address of the external scaler service. 
-        connection: <connection>                      # Mandatory. Connection string of Cosmos DB account with monitored container.
-        databaseId: <database-id>                     # Mandatory. ID of Cosmos DB database containing monitored container.
-        containerId: <container-id>                   # Mandatory. ID of monitored container.
-        leaseConnection: <lease-connection>           # Mandatory. Connection string of Cosmos DB account with lease container.
-        leaseDatabaseId: <lease-database-id>          # Mandatory. ID of Cosmos DB database containing lease container.
-        leaseContainerId: <lease-container-id>        # Mandatory. ID of lease container.
-        processorName: <processor-name>               # Mandatory. Name of change-feed processor used by listener application.
+        scalerAddress: external-scaler-azure-cosmos-db.keda:4050 # Mandatory. Address of the external scaler service.
```

Comment:
Maybe elaborate a bit on what the values mean for people who are not Kubernetes app/DNS experts?
```suggestion
        scalerAddress: external-scaler-azure-cosmos-db.keda:4050 # Mandatory. Address of the external scaler service, here our installation is called 'external-scaler-azure-cosmos-db' in the 'keda' namespace.
```

Reply:
Although it is mentioned below of course... Up to you!

Reply:
I would like to keep the comment concise to limit horizonal scrolling. The value of `scalerAddress` can be changed to `<scaler-name>:<scaler-namespace>:<port>` but think that would become more generic than what is required (that value will hold true for any scaler).